ido-vertical-mode.el

```diff
@@ -87,7 +87,7 @@ so we can restore it when turning `ido-vertical-mode' off")
 (defvar ido-vertical-count-active nil
   "Used internally to track whether we're already showing the count")
 
-(defcustom ido-vertical-define-keys 'C-n-and-C-p-only
```

Comment:
The usual trick here is to increase the major version of a software.  Then you can break everything ;-)

I don't understand why GitHub is showing me the commit that I made after I sent you this PR, instead of the original PR.  The last commit does not work!